functions/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "functions",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
```

Comment:
You need to install **firebase-admin** and **firebase-functions** as dependencies.
You need to install **firebase-functions-test** as a dev dependency.

Reply:
I have install these dependencies